package.json

```diff
@@ -131,8 +131,8 @@
     "resin-doodles": "0.0.1",
     "resin-image-fs": "^2.3.0",
     "resin-image-manager": "^5.0.0",
-    "resin-preload": "^5.0.0",
-    "resin-sdk": "^7.0.0",
+    "resin-preload": "beta",
+    "resin-sdk": "beta",
```

Comment:
We might reuse the `beta` tag in future for different SDK betas, so we shouldn't use that as our only specifier (otherwise, somebody installing this version of the CLI in the future could get a beta version of SDK 10, or anything).

`^9.0.0-beta` should work for the SDK (that'll match any SDK v9 releases, including all prereleases), and you can probably come up with something similar for resin-preload.

Reply:
updated to use sdk 9.0.0-beta and preload 6.0.0-beta